<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IrisDlg</class>
 <widget class="QDialog" name="IrisDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1021</width>
    <height>773</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Iris-Recognition</string>
  </property>
  <widget class="QGraphicsView" name="img_view">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>811</width>
     <height>691</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>0</y>
     <width>801</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_file_name">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>File Name:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_result">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Result：</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>837</x>
     <y>30</y>
     <width>161</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QRadioButton" name="radioBtn_resnet">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Resnet 34</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QRadioButton" name="radioBtn_transformer">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Transformer</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="btn_load">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Load Model</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="btn_import">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Import Image</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
